<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor jakości powietrza</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Dane stacji</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="fetchStationsButton">
         <property name="text">
          <string>Pobierz listę stacji</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="stationComboBox">
         <property name="placeholderText">
          <string>Wybierz stację...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sensorComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Wybierz czujnik...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fetchDataButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pobierz dane pomiarowe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Zarządzanie danymi</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveDataButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Zapisz dane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showChartButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Pokaż wykres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="analyzeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Analizuj dane</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="styleSheet">
       <string notr="true">font-weight: bold; color: #2c3e50;</string>
      </property>
      <property name="text">
       <string>Gotowy do pracy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="dataDisplay">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
